src/components/input/input.jsx

```diff
@@ -1,37 +1,65 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import cautionSvg from '../../svgs/icon-caution-fill.svg';
+import Icon from '../icon/icon.jsx';
 import styles from './input.css';
-import labelPropType from './label-prop-type';
 
-/**
- * Base input component. Accepts any prop that an input can take.
- * @see [RFC](https://github.com/mavenlink/rfc/pull/75)
- * @see [Design spec](https://zpl.io/295Yp7p)
- */
-export default function Input({ 'aria-labelledby': ariaLabelledBy, className, id, ...rest }) {
+function getLabelClassName(className, invalid) {
+  if (className) return className;
+  return invalid ? styles['invalid-label'] : styles.label;
+}
+
+function getClassName(className, invalid) {
+  if (className) return className;
+  return invalid ? styles['invalid-input'] : styles.input;
+}
+
+export default function Input(props) {
   return (
-    <input
-      {...rest}
-      aria-labelledby={ariaLabelledBy}
-      className={className}
-      id={id}
-    />
+    <React.Fragment>
```

Comment:
Why haven't we set up:

```jsx
<>
  <...>
</>
```

shorthand? I think it would be nice as `<React.Fragment>...</React.Fragment>` is super verbose

Reply:
I have the opposite opinion. I think `<>...</>` is unnecessary short-hand.

Reply:
What happens when you don't have this available and you upgrade a library if you don't keep up with the times? Well, you'll be temporarily stuck at `"react-styleguidist": "9.0.4"` for example:

https://github.com/styleguidist/react-styleguidist/issues/1321

I got same error as the result trying to use latest I believe because the library uses shorthand.

Reply:
If we do a bit more research, I think it comes with React. So the better question, why do we not have it already by just using React? Are we out-of-date with React? 

If we do have to do additional set up, it probably means the library is adding additional peer dependencies without documentation. Or their own dependency tree is broken.